lib/data/firebase_const.dart

```diff
@@ -2,16 +2,22 @@ const IMAGE_BASE_PATH = "/flamelink/media/sized";
 
 const FLAME_LINK_CONTENT = 'fl_content';
 const FLAME_LINK_SCHEMA = '_fl_meta_.schema';
+const FLAME_LINK_SCHEMA_TYPE = "_fl_meta_.schemaType";
 const FLAME_LINK_ENVIROMENT = '_fl_meta_.env';
 const FLAME_LINK_LOCALE = '_fl_meta_.locale';
 const PUBLICATION_STATUS = 'status';
 const DASH = '/';
 
+const String EMPTY = "";
 const String documentFieldContent = "content";
```

Comment:
Minor: What do we think about putting these in a class like DataStatus below, ready for when dart enums support templated types?
It could be:
class DocumentFields {
const String image = "image" 
.....
}

accessed via:  DocumentFields.image

Reply:
I'm not really sure if we can use enums as in Swift or Kotlin, but what you propose it's possible. Do you want me to change it in this PR? Or, as we talked, it's better to open a new branch to make this refactor?